repos/system_upgrade/el7toel8/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -95,7 +93,7 @@ def _get_active_kernel_modules(logger):
         # Use `modinfo` to probe for signature information
         parameter_dict = {}
         try:
-            signature = call(['modinfo', '-F', 'signature', name], split=False)
+            signature = run(['modinfo', '-F', 'signature', name])['stdout']
```

Comment:
I would keep the split explicitly when it is already set, but functionality is the same otherwise.
```suggestion
            signature = run(['modinfo', '-F', 'signature', name], split=False)['stdout']
```